### PR TITLE
bpo-33465: Use an unlikely to be built-in C extension in a test

### PR DESCRIPTION
**Pull request by warsaw**
https://bugs.python.org/issue33465

### CODE REVIEW

File: Lib/test/test_import/__init__.py [hunk 1]
```diff
@@ -91,12 +91,12 @@ def test_from_import_missing_attr_has_name_and_path(self):
         self.assertRegex(str(cm.exception), r"cannot import name 'i_dont_exist' from 'os' \(.*os.py\)")
 
     def test_from_import_missing_attr_has_name_and_so_path(self):
-        import select
+        import _hashlib
```
Comment (zware) [inline Lib/test/test_import/__init__.py]: It's pretty easy for `_hashlib` to just not exist, for instance if OpenSSL is not available.  How about `_testcapi`?

Comment (warsaw) [reply Lib/test/test_import/__init__.py]: Okay, after talking with @doko42 we think `_testcapi` is a good choice.  We need to find one that will always be built *and* is not a built-in (on Debian).

**Review by zware — APPROVED** [review]
I wonder if this test should be guarded by `@cpython_only`?  Otherwise, LGTM.

**Comment by warsaw** [comment]
@zware Good point.  Done.  And thanks for the approval; I'll merge this once CI passes.

File: Lib/test/test_import/__init__.py (hunk 1 above)
Comment (encukou) [reply Lib/test/test_import/__init__.py]: Same in Fedora; `_testcapi` sounds like a good choice.
(It's always built, as a shared library. Then it's packaged in `python3-tests` so it's not necessarily installed on all systems.)

**Pull request merged**